corehq/apps/users/models.py

```diff
@@ -1185,16 +1185,17 @@ def user_session_data(self):
         })
         return session_data
 
-    def delete(self, deleted_by, deleted_via=None):
+    def delete(self, domain, deleted_by, deleted_via=None):
```

Comment:
can't you get the domain from the object itself?

Reply:
For a CommCareUser you could, but not for a web user because they could have multiple domain memberships?